source/connect.txt

```diff
@@ -0,0 +1,25 @@
+.. _ruby-connect:
+
+==================
+Connect to MongoDB
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use the {+driver-short+} to connect to MongoDB.
```

Comment:
S: I don't think constants work in meta descriptions
```suggestion
   :description: Learn how to use the Ruby driver to connect to MongoDB.
```

Reply:
interesting.